### PR TITLE
Popup version project

### PR DESCRIPTION
**Pull request by maths29**
in this project we:
- implemented the popup version of the project to the 
- index.html and style.css, we also updated the js file using DOM to manipulate the UI interactions 
- we also added another js file for populating the content of the project section dynamically

### CODE REVIEW

File: display_portfolio.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+];
+
+const generateTechnologyTags = (technologies) => {
+  const html = technologies
+    .map((technology) => `<li>${technology}</li>`)
+    .join('\n');
+  return html;
+};
+
+// console.log(projectsData);
```
Comment (OyePriscilla) [inline display_portfolio.js]: - [ ] KIndly remove this commented code, this will make your codebase to be clean. ℹ️

File: display_portfolio.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+const projects = document.querySelector('#projects');
+const displayPopup = document.querySelector('#display-popup');
+let output = ' ';
+projectsData.forEach((project) => {
+  output += `
+        <div class="project-card">
+          <div class="project-img">
+            <img src="${project.featuredImage}" alt="">
+          </div>
```
Comment (OyePriscilla) [inline display_portfolio.js]: - [ ] Kindly add `alt` attribute to the cards images, the required alt attribute specifies an alternate text for an image, if the image cannot be displayed. ℹ️

File: display_portfolio.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  }
+});
+
+displayPopup.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (e.target.id === 'close-modal-popup') {
+    displayPopup.style.display = 'none';
+    document.body.style.overflow = 'auto';
+  }
+});
```
Comment (OyePriscilla) [inline display_portfolio.js]: - [ ] For your app to be according to the given design, kindly work more on the following: 

1. The `margins` around the pop-up.
2. The `spacing` between the two buttons.
3. The `hr` element is not bold as we have it the template.
4. Lastly, the stacks margin-bottom (`canopy, Back End Dev ...`)

@maths29 **Kindly take note that the same is applicable to the mobile version as well**

## Figma Design
Untitled

## Your Design
Untitled

## Figma Design
Untitled

## You Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @maths29 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Professional README file ✔️ 
Implemented pop-up modal ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @maths29, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**